.github/copilot-instructions.md

```diff
@@ -1,3 +1,18 @@
+# Project Overview
+
+This repo contains Azure SDKs for various Azure services and tools that generate them.
+
+## Folder Structure
+
+- `/eng/packages/http-client-csharp`: Contains the source code of the generator for Azure Data Plane SDKs (aka Azure Generator).
+- `/eng/packages/http-client-csharp-mgmt`: Contains the source code of the generator for Azure Management Plane SDKs (aka Azure Management Generator).
+- `/sdk`: Contains the individual SDKs for Azure services.
+
+### Azure Management Generator
```

Comment:
Can we add a corresponding section for the Azure Generator?

Reply:
yep will do that.
any suggestions?